src/app/components/Image/index.stories.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import Image from './index';
+
+const dataWithAltText = {
```

Comment:
I think this object should only include properties that the `Image` component consumes, so it is more parseable by humans :)

Reply:
Sure.